tests/testthat/test-examples.R

```diff
@@ -0,0 +1,127 @@
+# this test requires a `man` directory in the `tests/testthat` directory
+# (presumably symlinked to the package root `man` directory to avoid duplication)
+# this also requires `devtools::document()` to be run before running the tests
+
+rd_files <- function() {
+  list.files(testthat::test_path("man"), pattern = "\\.[Rr]d$", full.names = TRUE)
+}
+
+suppress_warnings <- function(expr, pattern = "*", ...) {
+  withCallingHandlers(
+    expr,
+    warning = function(w) {
+      if (grepl(pattern, conditionMessage(w))) {
+        invokeRestart("muffleWarning")
+      }
+    }
+  )
+}
+
+with_mocked_app_bindings <- function(code) {
+  shiny__shinyApp <- shiny::shinyApp # nolint object_name_linter.
+
+  # workaround of https://github.com/rstudio/shinytest2/issues/381
+  # change to `print(shiny__shinyApp(...))` and remove allow warning once fixed
+  mocked_shinyApp <- function(ui, server, ...) { # nolint object_name_linter.
+    functionBody(server) <- bquote({
+      pkgload::load_all(
+        .(normalizePath(file.path(testthat::test_path(), "../.."))),
```

Comment:
Minor detail as long as we're using `file.path` function here

```suggestion
        .(normalizePath(file.path(testthat::test_path(), "..", ".."))),
```